qualite/3-documenter-les-donnees.md

```diff
@@ -95,3 +95,29 @@ En pratique, il est souhaitable que le modèle de données et la nature de vos d
 
 ## Points de contact
 Les réutilisateurs des données peuvent avoir des questions à propos des fichiers mis à disposition. Il est conseillé de proposer un espace d’échange entre les producteurs et réutilisateurs des données. Il est préférable que cet espace d’échange soit public afin qu’il puisse bénéficier aux personnes qui auraient des questions similaires. De plus, la collecte des retours d’usage permettra d’améliorer votre documentation de manière incrémentale.
+
+## Le score de qualité des métadonnées  
+
+Nous avons mis en place sur [data.gouv.fr](http://data.gouv.fr/) un score de qualité des métadonnées pour répondre principalement aux deux problématiques suivantes :
+• D’une part, les réutilisateurs de données peinent à identifier les jeux de données de qualité et à évaluer si tel ou tel jeu de donnée est digne d’intérêt ;
+• D’autre part, les producteurs de données ne sont pas suffisamment incités et accompagnés à améliorer la qualité de leurs données.
```

Comment:
You should use `*` for a list with items. (See current result: https://guidesetalabgouv-qd1kqev72-etlb.vercel.app/qualite/documenter-les-donnees/#le-score-de-qualite-des-metadonnees)
```suggestion
* D’une part, les réutilisateurs de données peinent à identifier les jeux de données de qualité et à évaluer si tel ou tel jeu de donnée est digne d’intérêt ;
* D’autre part, les producteurs de données ne sont pas suffisamment incités et accompagnés à améliorer la qualité de leurs données.
```